recipes/e3sm_diags/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "e3sm_diags" %}
+{% set version = "2.2.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/e3sm_diags/archive/v{{ version }}.tar.gz
+  sha256: e1211d089a7857d11114071f38f099f25f3e6bb6d69d6c6e4e2cc1ccbb7b0dc0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - e3sm_diags = acme_diags.acme_diags_driver:main
+    - acme_diags = acme_diags.acme_diags_driver:main
+    - e3sm_diags_vars = acme_diags.acme_diags_vars:main
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+
+  run:
+    - python
+    - cdp
+    - cdms2
+    - cdutil
+    - genutil
+    - cdtime
+    - numpy
+    - matplotlib-base
+    - cartopy
+    - beautifulsoup4
+    - lxml
+    - dask 2.15.0
+
+test:
+  source_files:
+    - tests/*
+  imports:
+    - acme_diags
+  commands:
+    - e3sm_diags --help
+    - bash tests/test.sh
+
+about:
+  home: https://github.com/E3SM-Project/e3sm_diags
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: |
+    Diagnostics package for the DOE's Energy Exascale Earth System Model
+    (E3SM) project.
+  doc_url: https://e3sm-project.github.io/e3sm_diags/docs/html/index.html
+  dev_url: https://github.com/E3SM-Project/e3sm_diags
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - chengzhuzhang
+    - forsyth2
```

Comment:
@chengzhuzhang and @forsyth2, could you each confirm that you are willing to be a maintainer of this recipe

Reply:
Yes @xylar , I'm willing to be a maintainer.